.sphinx/lxddoc/lxddoc.txt

```diff
@@ -0,0 +1,43 @@
+---
+orphan: true
+---
+
+```{config:option} scheduler.instance cluster
+:liveupdate: `yes`
+:shortdesc: Possible values are all, manual and group. See Automatic placement of instances for more information.
+:condition: container
+:default: `all`
+:type: integer
+
+ The cluster config keys
```

Comment:
I can actually trim the spaces. I'll fix that.

Reply:
For the first line, we never want to start with spaces. But for other lines, we might.

So if this is indented because it is wrong in the Go file, that's okay - that's where we would need to fix it. But I can't see the input, so I'm not sure. :)